content/en/docs/quickstart.md

```diff
@@ -43,86 +43,96 @@ notation list --plain-http $IMAGE
 
 Confirm there are no signatures.
 
-## Generate a certificate
+## Generate a test key and self-signed certificate
 
-Use `notation cert generate-test` to generate a self-signed test certificate for signing artifacts. The following generates a self-signed X.509 certificate under the `~/config/notation/` directory.
+Use `notation cert generate-test` to generate a test RSA key for signing artifacts, and a self-signed test certificate for verifying artifacts.
 
 **IMPORTANT**: Self-signed certificates should be used for development purposes only and should not be used in production environments.
 
+The following command generates a test key and a self-signed X.509 certificate under the `$HOME/.config/notation/localkeys` directory. The test key is set as a default signing key using `--default` flag. The self-signed certificate is added to a named trust store `wabbit-networks.io` of type `ca`.
```

Comment:
If we use `$HOME/.config/notation/localkeys` here, I suggest adding a prerequisite to claiming that this Quick Start is using a Linux environment as an example. Since this path is different per OS.

Reply:
We can also add an inner link to [another document](https://github.com/notaryproject/notaryproject.dev/pull/80/files) that introduces the path when it is merged into the website.